packages/application/src/shell.ts

```diff
@@ -789,6 +792,34 @@ export class LabShell extends Widget implements JupyterFrontEnd.IShell {
     }
   }
 
+  /**
+   * Activate specified Area widget.
+   */
+  activateArea(area: ILabShell.Area = 'main'): void {
```

Comment:
Given that only `main` is implemented as of now I think we should add `@alpha` (is this the right one @fcollonval?) in documentation string and explain that only `main` is implemented.

Otherwise this PR looks fine to merge to me.

Reply:
@fcollonval @krassowski Shall I add @alpha to this comment now I would really like to get this merged then I can look at adding new elements once there is more to link to